cobalt/build/gn.py

```diff
@@ -51,10 +51,11 @@ def main(out_directory: str, platform: str, build_type: str,
 
 
 if __name__ == '__main__':
-  current_working_dir = os.getcwd()
-  python_env_var = os.getenv('PYTHONPATH')
+  current_working_dir = os.path.realpath(os.getcwd())
```

Comment:
This is a good change, hoever given that oscar is moving the check to GN: https://github.com/youtube/cobalt/pull/1536

changing it here is obsolete, you dont need the check in 2 places (python + GN)

@oxve  should ideally use this condition in his script.